tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -85,22 +85,22 @@
     <!--    Assembly: Grpc.AspNetCore.Server -->
     <!-- Integration: Grpc.Net.Client -->
     <!--    Assembly: Grpc.Net.Client -->
-    <!-- Latest package https://www.nuget.org/packages/Grpc.AspNetCore/2.65.0 -->
-    <PackageReference Include="Grpc.AspNetCore" Version="2.65.0" />
+    <!-- Latest package https://www.nuget.org/packages/Grpc.AspNetCore/2.66.0 -->
+    <PackageReference Include="Grpc.AspNetCore" Version="2.66.0" />
 
     <!-- Integration: HotChocolate.Execution -->
     <!--    Assembly: HotChocolate.Execution -->
-    <!-- Latest package https://www.nuget.org/packages/HotChocolate.AspNetCore/13.9.11 -->
-    <PackageReference Include="HotChocolate.AspNetCore" Version="13.9.11" />
+    <!-- Latest package https://www.nuget.org/packages/HotChocolate.AspNetCore/13.9.12 -->
+    <PackageReference Include="HotChocolate.AspNetCore" Version="13.9.12" />
 
     <!-- Integration: amqmdnetstd -->
     <!--    Assembly: amqmdnetstd -->
-    <!-- Latest package https://www.nuget.org/packages/IBMMQDotnetClient/9.4.0 -->
-    <PackageReference Include="IBMMQDotnetClient" Version="9.4.0" />
+    <!-- Latest package https://www.nuget.org/packages/IBMMQDotnetClient/9.4.0.5 -->
+    <PackageReference Include="IBMMQDotnetClient" Version="9.4.0.5" />
 
     <!-- Integration: log4net -->
     <!--    Assembly: log4net -->
-    <!-- Latest package https://www.nuget.org/packages/log4net/2.0.17 -->
+    <!-- Latest package https://www.nuget.org/packages/log4net/3.0.0 -->
     <PackageReference Include="log4net" Version="2.0.17" />
```

Comment:
We need to add support here for a new major version: @bouwkast @lucaspimentel

Reply:
Noted, I'll get a PR going for it